helper/restore_helper.go

```diff
@@ -138,7 +154,26 @@ type oidWithBatch struct {
 	batch int
 }
 
+type IRestoreHelper interface {
+	getOidWithBatchListFromFile(oidFileName string) ([]oidWithBatch, error)
+	flushAndCloseRestoreWriter(pipeName string, oid int) error
+	closeAndDeletePipe(tableOid int, batchNum int)
+	getRestoreDataReader(fileToRead string, objToc *toc.SegmentTOC, oidList []int) (IRestoreReader, error)
+	getRestorePipeWriter(currentPipe string) (*bufio.Writer, *os.File, error)
+	checkForSkipFile(pipeFile string, tableOid int) bool
+}
+
+type RestoreHelper struct{}
+
+func (RestoreHelper) checkForSkipFile(pipeFile string, tableOid int) bool {
+	return utils.FileExists(fmt.Sprintf("%s_skip_%d", pipeFile, tableOid))
+}
+
 func doRestoreAgent() error {
+	return doRestoreAgentInternal(new(Helper), new(RestoreHelper))
+}
+
+func doRestoreAgentInternal(h IHelper, rh IRestoreHelper) error {
```

Comment:
I have doubts that `h` and `rh` are good namings here. Short names are good for objects with very local scope. But these are used thourhout the long `doRestoreAgentInternal()` function.  I've been reviewing the function for a few times now, and each time I "stumble" when I see references to `h` or `rh`, and need to go to the function start to refresh my context. 
I suggest to rename them at least here to smth like `helper` and `restoreHelper` or similar.

Reply:
good point!